### DISCUSSION TITLE
Issues with Google Analytics Integration on GitBook

### DISCUSSION THREAD

**Discussion by sid-raga** [GitbookIO/community#463] [0. General bugs and questions] [answered]
I have followed the procedure for integrating Google Analytics by pasting the tracking ID into the appropriate section. After this I was able to track activity on the Google Analytics Dashboard. However, after publishing into a custom domain (https://docs.raga.ai/) I am not observing any events being logged in my Google Analytics dashboard. 

Can anyone please provide guidance or insights on what might be going wrong?

**Comment by aikoii** [comment, marked as answer]
👋 @sid-raga I'm looking at your space now, and the tracking ID you've added doesn't seem correct. It should be something like UA-XXXXX-Y or G-XXXXXXX.

**Reply by adamkitain** [reply]
Uninstalling and re-installing the GA Integration fixed it for me.

**Reply by aikoii** [reply]
That's great to hear! 🌟
